spec/numberformat.html

```diff
@@ -618,6 +618,23 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-intl.numberformat.prototype.formattoparts">
+      <h1>Intl.NumberFormat.prototype.formatToParts ( [ _value_ ] )</h1>
```

Comment:
Thanks. Should there be an explicit check for the existence of the argument, too?